### PR TITLE
Add breaks for exceptions to match Step 2 of Ui in DG

### PR DESCRIPTION
**Pull request by nikkiDEEE**
Added the try catch (done using breaks because seq diag no try-catch, they just use breaks) for exceptions to better represent the steps 1-3 for DG under Ui implementation

### CODE REVIEW

File: docs/sequence-diagrams/diagrams/Ui.png
Comment (heejet) [inline docs/sequence-diagrams/diagrams/Ui.png:1]: When an exceptions occurs, the `Ui` calls its own `printReadFail()` method not the Scanner.

**Review by heejet — CHANGES_REQUESTED** [review]

**Comment by heejet** [comment]
U can refer to this also https://github.com/nus-cs2113-AY2021S2/forum/issues/53

**Comment by nikkiDEEE** [comment]
> U can refer to this also [nus-cs2113-AY2021S2/forum#53](https://github.com/nus-cs2113-AY2021S2/forum/issues/53)

got it, thanks! since there isnt much nesting for mine, i went with the alt statement instead of using reference frames to explain the printReadFail method being called from the exceptions

**Comment by heejet** [comment]
> > U can refer to this also [nus-cs2113-AY2021S2/forum#53](https://github.com/nus-cs2113-AY2021S2/forum/issues/53)
> 
> got it, thanks! since there isnt much nesting for mine, i went with the alt statement instead of using reference frames to explain the printReadFail method being called from the exceptions
> 
> should i add `note over Ui` for the 2 exceptions stating `fullCommand == EXIT_COMMAND` is returned?

Yes that would be better

File: docs/sequence-diagrams/diagrams/HelpFeature.png
[the remark this answers is not in the document]
Comment (nikkiDEEE) [reply docs/sequence-diagrams/diagrams/HelpFeature.png:1]: I have made the changes for this in the latest push to this PR

**Comment by heejet** [comment]
> 
 I think your current diagram is fine just that you forgot to put in the type. `FullUserInput` should be `FullUserInput: String` and `EXIT_COMMAND` should be `EXIT_COMMAND: String`

**Comment by nikkiDEEE** [comment]
> > 
> 
> I think your current diagram is fine just that you forgot to put in the type. `FullUserInput` should be `FullUserInput: String` and `EXIT_COMMAND` should be `EXIT_COMMAND: String`

yup added

**Review by heejet — APPROVED** [review]

**Review by heejet — APPROVED** [review]
Looks good to merge after resolving conflicts

**Pull request merged**